### PR TITLE
Automatically build brew bottle

### PR DESCRIPTION
**Pull request by dwightguth**

### CODE REVIEW

**Comment by dwightguth** [comment]
My mistake, this is not actually ready yet.

File: Jenkinsfile [hunk 1]
```diff
@@ -158,13 +158,16 @@ pipeline {
           }
         }
       }
-    }
+    }*/
```
Comment (ehildenb) [inline Jenkinsfile]: remove comment tags

File: Jenkinsfile [hunk 2]
```diff
@@ -158,13 +158,16 @@ pipeline {
...
     stage('Release') {
       agent { label 'docker' }
       options { skipDefaultCheckout() }
       environment {
         GITHUB_TOKEN    = credentials('rv-jenkins')
         KEVM_RELEASE_ID = '1.0.0'
+        PACKAGE = 'kevm'
+        VERSION = '1.0.0'
+        ROOT_URL = 'https://github.com/kframework/evm-semantics/releases/download/nightly'
```
Comment (ehildenb) [inline Jenkinsfile]: Is there a reason for `VERSION` separate from `KEVM_RELEASE_ID`?

Comment (ehildenb) [reply Jenkinsfile]: It doesn't seem to be used directly anywhere here.

File: Jenkinsfile [hunk 3]
```diff
@@ -235,11 +238,11 @@ pipeline {
           }
         }
         stage('Deploy Release') {
-          when {
+/*          when {
             not { changeRequest() }
             branch 'master'
             beforeAgent true
-          }
+          }*/
```
Comment (ehildenb) [inline Jenkinsfile]: remove comment tags

File: tests/node-firefly [hunk 4]
```diff
@@ -27,11 +27,10 @@ cd openzeppelin-solidity
...
 npm install
-npx truffle compile
 
 # Wait for kevm-vm to start up
 while ! netcat -z localhost "$kevm_vm_port"; do
     sleep 1
 done
 
-npx truffle test
+node_modules/.bin/truffle test
```
Comment (ehildenb) [inline tests/node-firefly]: Why these changes?

File: Jenkinsfile (hunk 2 above)
Comment (dwightguth) [reply Jenkinsfile]: This is used by the brew scripts that we are invoking that are part of the kframework package. They are used to control the CI so that it will test the correct commit on Mac OS without deploying it first.

File: tests/node-firefly (hunk 4 above)
Comment (dwightguth) [reply tests/node-firefly]: So that we always use the version of truffle specified by the package rather than the latest version which may not be backwards compatible.
